libs/judicial-system/formatters/src/lib/formatters.ts

```diff
@@ -138,6 +139,24 @@ export const getHumanReadableCaseIndictmentRulingDecision = (
       return 'Ekki skráð'
   }
 }
+export const getRoleTitleFromCaseFileCategory = (
```

Comment:
Please consider moving this method declaration to the web project as it is only used there.

Reply:
Does it have to be only for formatters that are shared between projects? At least in this case the components that depend on this formatter are in different contexts within web so I think its makes it cleaner to keep it here, wdyt?

Reply:
Generally, I have tried to keep code that is only used within one project in that project rather than in a lib. That way we know that libs contain (mostly) shared code, which is kind of the purpose of the libs. However, there are exceptions and I will not challenge this further if you really want to keep the method here 😅